projects/sport-data-valley/manual_seeds/physical_complaints_responses.rb

```diff
@@ -153,7 +153,7 @@ def create
 
         date = START_DATE
         while date <= END_DATE
-          create_rheuma_responses(person, date)
+          create_ostrc_h_o_responses(person, date)
```

Comment:
same here, was this wrong in the seeds?

Reply:
Yeah this was wrong

Reply:
I already changed this on our end months ago to fix the seeding for weekly complaints questionnaire. The seeding of this questionnaire was broken before the updates

Reply:
I will remove these changes (unrelated to updating for the BMX option) and create a separate PR